src/test/java/com/zendesk/maxwell/MaxwellTestSupport.java

```diff
@@ -221,7 +221,7 @@ public void run() {
 		}
 
 		callback.afterReplicatorStart(mysql);
-		maxwell.context.getBootstrapController(null).interrupt();
+		BootstrapController bootstrapController = maxwell.context.getBootstrapController(null);
```

Comment:
`bootstrapController` is never used.  Can this call be removed (or at least the variable assignment)?